roles/keystone/tasks/federation.yml

```diff
@@ -10,8 +10,10 @@
   retries: 5
 
 - name: download mod_auth_openidc
-  get_url: url={{ keystone.federation.sp.oidc.download.url }}
-           dest=/tmp/{{ keystone.federation.sp.oidc.module_name }}
+  get_url:
+    url: "{{ keystone.federation.sp.oidc.download.url }}"
+    dest: /tmp/{{ keystone.federation.sp.oidc.module_name }}
+    checksum: "{{ keystone.federation.sp.oidc.download.checksum }}"
```

Comment:
This is the previous PR: https://github.com/blueboxgroup/ursula/pull/2058
If I understand your question correctly, I think that there is no way to use apt/debian url with a checksum (which sounds like the same thing as downloading as installing). http://docs.ansible.com/ansible/apt_module.html (no checksum parameter).

I guess the other option would be to add this package into some repo and have it installed through apt (Which I assume package signing would be part of that solution). I'm not sure how that would work though.